### ISSUE TITLE
Increased amyloid beta 42 isoform level

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7071]
**Preferred term label:**
Increased amyloid beta 42 isoform level 

**Synonyms**
Increased beta amyloid 42 level, Increased Abeta 42 level

**Definition (free text, please give PubMed ID)**
Higher than normal level of amyloid beta 42 found in cerebrospinal fluid.
PMID:31391004

**Parent term (use hpo.jax.org/app)**
Abnormal amyloid beta 42 isoform level 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 607822

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
@sarahstofel -- I am not an expert but have found literature that beta-amyloid(1-42)  is decreased in AD. I have not found any literature about a clinically relevant increase in beta-amyloid(1-42) in any disease and did not see this in the OMIM entry you referenced above. Therefore my question -- is it sufficient to have a "decreased" term or do we also need the "increased" term? If the latter, could you please provide a PMID?

https://jamanetwork.com/journals/jamaneurology/fullarticle/775090

Comment: Beta-amyloid is generated continuously as a soluble protein during normal cellular metabolism and is secreted into the extracellular space and biological fluids and thus, into the CSF. There are 2 major C-terminal variants of beta-amyloid: a shorter form ending at Val-39 or Val-40 and a longer form ending at Ala-42. The different forms of amyloid deposits contain different C-terminally truncated forms of β-amyloid, with beta-amyloid(1-42) predominating in diffuse plaques, SP cores, and cortical homogenate; and shorter forms predominating in vascular amyloid and CSF. CSF beta-amyloid(1-42) has been shown to be reduced in patients with Alzheiumer disease compared with controls. PMID:10369305

**Comment by pnrobinson** [comment]
Also, OMIM:619405  2/2   PMID:31778857  CSF examination showed low Aβ1–42 (363 ng/l; n.v. > 550), similar to that seen in idiopathic AD (Motter et al, 1995; Andreasen et al, 1999)

**Comment by pnrobinson** [comment]
added term. Leaving question of AD annotations open for now, please recontact if there is more information.
